dataprofiler/data_readers/graph_data.py

```diff
@@ -12,9 +13,14 @@
 class GraphData(BaseData):
     """GraphData class to identify, read, and load graph data."""
 
-    data_type = "graph"
+    data_type: Optional[str] = "graph"
```

Comment:
in this file... shouldn't this always be `data_type: str`?

Reply:
The issue is that `BaseData` sets the field to `None`.